lighthouse-core/lib/asset-saver.js

```diff
@@ -326,13 +326,16 @@ async function saveAssets(artifacts, audits, pathWithBasename) {
 async function logAssets(artifacts, audits) {
   const allAssets = await prepareAssets(artifacts, audits);
   allAssets.map(passAssets => {
-    log.log(`devtoolslog-${passAssets.passName}.json`, JSON.stringify(passAssets.devtoolsLog));
+    const dtlogdata=JSON.stringify(passAssets.devtoolsLog);
```

Comment:
spaces?

Reply:
nah it's more straightforward without any extra whitespace

Reply:
> nah it's more straightforward without any extra whitespace

uh...wha?

Reply:
OHHH i thought the suggestion was to add a spaceindent arg to stringify. :)  🤡